sycl/doc/extensions/experimental/sycl_ext_intel_fpga_task_sequence.asciidoc

```diff
@@ -0,0 +1,315 @@
+
+== `task_sequence` Properties
+
+The following code and table describe the "balanced", "invocation_capacity", and "response_capacity" properties that can be provided when declaring a `task_sequence`.
+
+[source,c++,linenums]
+----
+namespace sycl {
+namespace ext {
+namespace intel {
+namespace experimental {
+
+struct balanced {
+  using value_t = property_value<balanced>;
+};
+
+struct invocation_capacity {
+  template <uint32_t Size>
+  using value_t = property_value<invocation_capacity, std::integral_constant<uint32_t, Size>>;
+};
+
+struct response_capacity {
+  template <uint32_t Size>
+  using value_t = property_value<response_capacity, std::integral_constant<uint32_t, Size>>;
+};
+
+inline constexpr balanced::value_t balanced_v;
+
+template <uint32_t Size>
+inline constexpr invocation_capacity::value_t<Size> invocation_capacity_v;
+
+template <uint32_t Size>
+inline constexpr response_capacity::value_t<Size> response_capacity_v;
+
+} // namespace experimental
+} // namespace intel
+} // namespace ext
+} // namespace sycl
+----
+
+--
+[options="header"]
+|===
+| Property | Description
+| balanced | The `balanced` property is a guarantee to the SYCL device compiler that the `task_sequence` object will call exactly the same number of `async` s and `get` s over the object's lifetime (i.e. before the `task_sequence` destructor is invoked). 
+
+| invocation_capacity | The `invocation_capacity` property defines the minimum required number `Size` of `async` invocations that must be invokable without any `get` being called on that same set.  This number of `async` invocations without a `get` call is the minimum number that will be supported before the `async` member function blocks.
+
+| response_capacity | The `response_capacity` property defines the maximum number `Size` of outstanding `async` invocations such that all outstanding invocations are guaranteed to make forward progress. Further `async` invocations beyond `Size` may block until enough `get` calls are invoked such that the number of outstanding `async` invocations is reduced to `Size`.
+|===
+--
+
+=== Compatibility with FPGA Kernel Interface Properties
+
+The Callable `f` defining the asynchronous task associated with a `task_sequence` may be declared with the FPGA Kernel Interface `pipelined` property described in link:../proposed/sycl_ext_oneapi_kernel_properties.asciidoc[sycl_ext_oneapi_kernel_properties].
+
+
```

Comment:
Can you add a section here with simple code samples? Usually the extensions for new classes show how the class will be used.

Reply:
Added a section.